docker/cron/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.fedoraproject.org/fedora:38
+MAINTAINER copr-devel@lists.fedorahosted.org
+
+RUN dnf -y update
+RUN dnf -y install cronie tar
+
+RUN touch /var/log/cron.log
+
+COPY files/cron/tar_persistent.sh /bin/tar_persistent
+RUN chmod +x /bin/tar_persistent
```

Comment:
I think you can `chmod +x` the `files/cron/tar_persistent.sh` script and then commit it into the git repo. Two advantages:

1. We won't need to run `chmod +x` in the Dockerfile
2. We will be able to run the script even outside of the container

Reply:
it seems like COPY doesn't preserve permissions so I specified chmod directly to COPY command